orc8r/cloud/deploy/orc8r_deployer/docker/root/scripts/cli/configlib.py

```diff
@@ -228,7 +231,14 @@ def initialize_defaults(self, component):
 
                 # if defaults are overriden already, explicitly confirm
                 # to reset defaults
-                if (not curr_val or (curr_val and default != curr_val and
-                    click.confirm(f"Override {config_key} "
-                    f"current val {curr_val} with default {default}"))):
+                if not curr_val:
                     cfgs[config_key] = default
+
+                if override_defaults:
+                    continue
+
+                if curr_val and default != curr_val:
+                    if click.confirm(f"Override {config_key} "
+                        f"current val {curr_val} with default {default}"):
+                        cfgs[config_key] = default
+                    
```

Comment:
Here and elsewhere in PR, there are some whitespace linting warnings. Can you quickly fix them up?